### ISSUE TITLE
RethinkDB shutting down

### ISSUE THREAD

**Issue by chrbala** [mattkrick/meatier#173]
https://rethinkdb.com/blog/rethinkdb-shutdown/

What are your thoughts? Think the community will take it over? Is there a good replacement that will still be developed?

**Comment by patrickleet** [comment]
https://goo.gl/forms/WnJztEIEx8Ivkrwk2 everyone fill this out! Rethinkdb
needs a new home!

**Comment by mattkrick** [comment]
@chrbala RethinkDB (the technology) isn't going anywhere. ReQL is the most beautiful DB language I've ever used (JS bias acknowledged). The docs are the GOLD STANDARD for how docs should be written. No DB admin console (localhost:8080) even comes close to what they offer. Sharding across table joins, built-in pub/sub, etc are just icing on the cake.

OSS is a strange field. Things like Node.js are owned by the Linux Foundation & do just fine. Other things like Django are completely community run where folks use kickstarter to fund their PRs. Postgres is somewhere in the middle I think, I never really knew where $$/PRs came from for them.

At work, we use RethinkDB in production & have no desire to switch. Even if another PR is never made, it's stable, performant, and offers all the features we need.

**Comment by zalmoxisus** [comment]
RethinkDB is a mature product, also Slava, Mike and all the team will use it in production at Stripe, so it could get even better (in terms of optimizations, not features). Just Horizon could stay without future.

**Comment by chrbala** [comment]
Yes, I was thinking the same @mattkrick. I really hope the community can build it to something even better though!

**Comment by bartekus** [comment]
The community is now actually doubling in strength upon hearing about this unfortunate situation and many outside entities involved in OSS now step forward to support the transition, upkeep and continuous evolution of both RethinkDB and Horizon. Things are actually looking very promising. Even proper legal team has been assembled to deal with relicensing and any potential troubles that could occur during this process. RethinkDB and Horizon are here to stay and on top of it, I'm also working on bringing fusetools.com to the fold as they are looking exactly for this type of solution for their own platform.

**Comment by quadsurf** [comment]
@mattkrick I think the following would be an amazing stack to work with:
meatier, relay instead of redux, horizon.io, graphQL, rethinkDB

**Comment by mattkrick** [comment]
@quadsurf You'll have to pick between horizon & relay. Relay (both v1 and v2) doesn't support subscriptions, so using that with horizon is gonna be rough, unless you want to keep 2 stores. You could use apollo, but then you have to either write an in-memory MQ or build a wrapper around a very robust one (i'd suggest rabbitMQ). Then, you have to write event wrappers & awkward n+1 subscribers...

Personally, I use Cashay, which is something I wrote. I gave extra care in making subscriptions super easy.
For example, if you want to get every person on your team & every project that they own:
```
query {
  teamMembers (teamId: $teamId) @live {
    id
    preferredName,
    projects @live {
      content
      id
    }
  }
}
```
That's pulled straight out of a project that's running in production. Try doing that with any other client cache :wink:

**Comment by patrickleet** [comment]
I like the live feature. I've been using Apollo but this makes me
interested in checking out cashay.

As far as good wrappers around rabbitmq, I use servicebus(
https://github.com/mateodelnorte/servicebus) for microservices.

**Comment by chrbala** [comment]
Good news! Rethinkdb is now a part of the Linux Foundation!

https://www.linux.com/news/2017/2/rethinkdbs-realtime-cloud-database-lands-linux-foundation

**Comment by dmitry** [comment]
@chrbala thanks for sharing. Very good news!
